dev/kyuubi-extension-spark-3-1/src/test/scala/org/apache/spark/sql/KyuubiExtensionSuite.scala

```diff
@@ -1584,4 +1584,40 @@ class KyuubiExtensionSuite extends QueryTest with SQLTestUtils with AdaptiveSpar
       checkZorderTable(true, "c4, c2, c1, c3", false, true)
     }
   }
+
+  test("test watchdog with scan maxHivePartitions") {
+    withTable("test", "temp") {
+      sql(
+        s"""
+           |CREATE TABLE test(i int)
+           |PARTITIONED BY (p int)
+           |STORED AS textfile""".stripMargin)
+      spark.range(0, 10000, 1).selectExpr("id as col")
```

Comment:
10 rows is enough, otherwise we will produce 10000 * 10 rows data ..

Reply:
> 10 rows is enough, otherwise we will produce 10000 * 10 rows data ..

Excessive force～